src/vs/editor/common/model/tokenizationTextModelPart.ts

```diff
@@ -59,12 +59,6 @@ export class TokenizationTextModelPart extends TextModelPart implements ITokeniz
 	) {
 		super();
 
-		this._register(this._languageConfigurationService.onDidChange(e => {
-			if (e.affects(this._languageId)) {
-				this._onDidChangeLanguageConfiguration.fire({});
-			}
-		}));
-
 		// We just look at registry changes to determine whether to use tree sitter.
```

Comment:
Should be safe to because the same event is listened on in the `TextModel`:

https://github.com/microsoft/vscode/blob/df257e194e465bf229f7f8863efc62c7760f2af3/src/vs/editor/common/model/textModel.ts#L386-L389

Which calls `handleLanguageConfigurationServiceChange`, which does the same exact thing:

https://github.com/microsoft/vscode/blob/df257e194e465bf229f7f8863efc62c7760f2af3/src/vs/editor/common/model/tokenizationTextModelPart.ts#L115-L119